### ISSUE TITLE
Focus styles are missing for checkbox and radio

### ISSUE THREAD

**Issue by luminarious** [ireade/formhack#2]
Overall it looks good, but on Chrome, I noticed the distinct lack of keyboard focus styles.

**Comment by ireade** [comment]
Thanks for letting me know, I just made an update to fix this
